src/App.css

```diff
@@ -1,3 +1,269 @@
 .App {
+  background-color: #fafafa;
+  font-family: Montserrat, sans-serif;
+}
+
```

Comment:
- [ ] Your font families are not rendering correctly even if they are declared because they haven't been imported to the project.  Please import your fonts so they actually renter on the page.

## Hints

- Go to [Montserrat font](https://fonts.google.com/specimen/Montserrat), and [Roboto Slab](https://fonts.google.com/specimen/Roboto+Slab).
- Select all the weights that you need.
- Copy the import rule that results.
- I did this for you:

Add this to the beginning of your CSS file:

```css
@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600;700&display=swap');
@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;300;400;500;600;700&display=swap');
```

That way, both fonts will be imported 💪

Reply:
Thank you